packages/docteur-unix/docteur-unix.0.0.2/opam

```diff
@@ -11,7 +11,7 @@ depends: [
   "ocaml" {>= "4.07.0"}
   "dune" {>= "2.8.0"}
   "docteur" {= version}
-  "mirage-unix" {>= "4.0.0"}
+  "mirage-unix" {>= "4.0.0" & < "5.0.0"}
```

Comment:
I still don't understand why we need an upper bound here. What error is this fixing?

Reply:
ok got it, mirage3 needs mirage-unix.4 and mirag4 needs mirage-unix.5